flask_resty/filtering.py

```diff
@@ -12,10 +13,7 @@
 # -----------------------------------------------------------------------------
 
 # Field.missing is deprecated in favor of Field.load_default in marshmallow 3.13.0
-try:
-    _USE_LOAD_DEFAULT = marshmallow.__version_info__ >= (3, 13)
-except AttributeError:
-    _USE_LOAD_DEFAULT = True
+_USE_LOAD_DEFAULT = importlib.metadata.version("marshmallow")
```

Comment:
this doesn't looks quite right. still need to check the version number here

Reply:
🤦🏼 should be fixed now.